src/domains/dashboard/components/node-view/use-update-theme.ts

```diff
@@ -0,0 +1,64 @@
+import { useContext, useMemo } from "react"
+import { ThemeContext } from "styled-components"
+
+export const useUpdateTheme = () => {
+  const theme = useContext(ThemeContext)
+  const currentTheme = window.NETDATA.themes.current
+
+  useMemo(() => {
+    if (!theme.colors) return
+
+    window.NETDATA.themes.current = {
+      ...currentTheme,
+      background: theme.colors.elementBackground,
+      foreground: theme.colors.border,
+      grid: theme.colors.disabled,
+      axis: theme.colors.elementBackground,
+      highlight: theme.colors.attention,
+      colors: [
+        "#B0E952",
```

Comment:
I suppose these match the theme colours used in the cloud. So couldn't we reference them from UIKit same way the cloud does? (`getColor("color-name...")` etc)?

Reply:
Those are new actually, just for the charts. We can completely update the charts colors in the ui kit and use them here directly in the future.